charts/ontology/README.md

```diff
@@ -0,0 +1,79 @@
+DUOS Ontology
+========
+A Helm chart for DUOS Ontology, the DUOS Algorithmic Matching System
+
+Current chart version is `0.1.0`
+
+
+
+
+
+## Chart Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| global.applicationVersion | string | Is set by Helmfile on deploy | Ontology global version |
+| appVersion | string | Is set by Helmfile on deploy | Ontology image version/tag. Required unless using `image`. |
```

Comment:
Please use `global.applicationVersion` instead of `appVersion`. (Sorry, I know this is confusing and not documented anywhere...)

Our initial terra-helmfile implementation supplied a value called `appVersion` to all app charts but we ended up changing that to a global value so that it could be consumed by subcharts. For now, terra-helmfile supplies both values, but `global.applicationVersion `is the correct value to use, `appVersion` is deprecated. All Terra app charts that have a value called `appVersion` should rename the value to `global.applicationVersion`.